xmlserializer/XmlElement.cpp

```diff
@@ -271,3 +262,18 @@ bool CXmlElement::CChildIterator::next(CXmlElement& xmlChildElement)
     return false;
 }
 
+template bool CXmlElement::getAttribute(const std::string &name, std::string &value) const;
+template bool CXmlElement::getAttribute(const std::string &name, uint16_t &value) const;
+template bool CXmlElement::getAttribute(const std::string &name, uint32_t &value) const;
+template bool CXmlElement::getAttribute(const std::string &name, int32_t &value) const;
+template bool CXmlElement::getAttribute(const std::string &name, uint64_t &value) const;
```

Comment:
I think you forget a couple:
 - bool
 - {unsigned,signed,} char
 - {unsigned,} short
 - {unsigned,} int
 - {unsigned,} long
 - {unsigned,} long long
 - float
 - {long,} double
 - std::string